sema/syncfs/service.py

```
@@ -21,6 +22,22 @@
 DEFAULT_URN_BASE = "urn:sync:"
 
 
+class SyncFsResult(ServiceResult):
+    """Result of the syncfs service"""
+
+    def __init__(self) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)